enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -23,7 +23,25 @@ Features planned:
 
 # Getting started
 
-## Start a local build of Metabase from master
+## Start Metabase
+
+You have a couple of options to start Metabase:
+
+### 1. Running on Docker
+Start the Metabase container:
+```bash
+docker run -d -p 3000:3000 --name metabase metabase/metabase-dev:embedding-sdk-0.1.0
+```
+
+### 2. Running the Jar file
+1. Download the Jar file from http://downloads.metabase.com/sdk/v0.1.0/metabase.jar
```

Comment:
I use the copy from https://www.metabase.com/docs/latest/installation-and-operation/running-the-metabase-jar-file